src/app/app.component.html

```diff
@@ -1,7 +1,11 @@
-<alm-app-header></alm-app-header>
-
-<main class="app">
-  <router-outlet></router-outlet>
+<header class="padding-bottom-10" >
```

Comment:
I've created an issue for this, so we can come back to it. I suggest we just accept the change for now.